_api-reference/cat/cat-segment-replication.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+title: CAT segment replication
+parent: CAT API
+nav_order: 53
+has_children: false
+---
+
+# CAT segment replication
+Introduced 2.7
+{: .label .label-purple }
+
+The CAT segment replication operation returns information about active and last completed [segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/index) events on each replica shard, including related shard-level metrics. These metrics provide information about how far behind the replicas are lagging with respect to the primary shard. 
```

Comment:
Last sentence: "These metrics provide information about how far behind the primary shard the replicas are lagging"? As written, this may be initially read as though something is lagging behind the replicas.